modmesh/app/euler1d.py

```diff
@@ -78,6 +81,8 @@ class QuantityLine:
     ana: matplotlib.lines.Line2D = None
     num: matplotlib.lines.Line2D = None
     axis: matplotlib.pyplot.axis = None
+    y_upper_lim: float = 0.0
+    y_bottom_lim: float = 0.0
```

Comment:
added 2 float as Quantity Line's attribute to record current data line y axis upper / bottom limit.